LaunchScreenDemo/WebServices/Alamofire/Controller/JeweleryUpdateViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  JeweleryUpdateViewController.swift
+//  LaunchScreenDemo
+//
+//  Created by Vidhi Patel on 06/07/23.
+//
+
+import UIKit
+
+class JeweleryUpdateViewController: UIViewController {
+    
+    //MARK: - IBOutlets and Vars
+    @IBOutlet private weak var descriptionLbl: UILabel!
+    @IBOutlet private weak var updateBtn: UIButton!
+    @IBOutlet private weak var priceLbl: UILabel!
+    @IBOutlet private weak var titleLbl: UILabel!
+    @IBOutlet private weak var imageView: UIImageView!
+    var product: ProductRequestModel?
+    private var viewModel: JeweleryViewModel = JeweleryViewModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setData()
+    }
+    
+    //MARK: - Private methods
+    private func setData() {
+        if let url = URL(string: product?.image ?? "") {
+            URLSession.shared.dataTask(with: url) { (data, response, error) in
+                guard let images = data else {
+                    return
+                }
+                DispatchQueue.main.async {
+                    self.imageView.image = UIImage(data: images)
+                }
+            }.resume()
+        }
+        titleLbl.text = product?.title
+        priceLbl.text = "$\(product?.price ?? 0)"
+        descriptionLbl.text = product?.description
+    }
+
+    @IBAction func updateBtnAction(_ sender: UIButton) {
+        viewModel.updateProduct(id: 0, title: "", image: "", price: 0, description: "", category: "", rating: RatingUpdate(rate: 0, count: 0))
```

Comment:
can you please explain this statement?  no data passed in model for update ?

Reply:
Because the FakeStore API don't update the data but we have to pass data to call API.

Reply:
<img width="921" alt="Screenshot 2023-07-07 at 5 01 43 PM" src="https://github.com/vidhi48/Simform-iOS/assets/99391641/f0d5e009-f477-45f7-849c-a0de3a27d9f7">
I can see in doc there is actual data not blank data and you are pasing blank data instead of actual data please uodate accordingly

Reply:
Updated data.